Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -45,11 +45,33 @@ public class GraphLinter: GraphLinting {
         issues.append(contentsOf: lintMismatchingConfigurations(graphTraverser: graphTraverser))
         issues.append(contentsOf: lintWatchBundleIndentifiers(graphTraverser: graphTraverser))
         issues.append(contentsOf: lintCodeCoverageMode(graphTraverser: graphTraverser))
+        issues.append(contentsOf: lintWorkspaceUnuesedTargetsInWorkspace(graphTraverser: graphTraverser))
         return issues
     }
 
     // MARK: - Fileprivate
 
+    private func lintWorkspaceUnuesedTargetsInWorkspace(graphTraverser: GraphTraversing) -> [LintingIssue] {
+        graphTraverser.workspace.schemes.map { scheme in
+            let arr = scheme
+                .targetDependencies()
+                .filter { targetReference in
+                    if let target = graphTraverser
+                        .targets()[targetReference.projectPath],
+                        target.keys.contains(targetReference.name)
+                    {
+                        return false
+                    }
+                    return true
+                }
+                .map(\.name)
+            return LintingIssue(
+                reason: "Cannot find targets \(arr.joined(separator: ", ")) defined in \(scheme.name)",
```

Comment:
is it worth including the relative project path as well? 

e.g. in a scenario where an incorrect path is specified:

```swift
.testableTarget(target: .project(
    path: "Frameworks/Framework1_incorrect_path",
    target: "Framework1Tests"
)),
```

The error would indicate:

```
 · Cannot find targets Framework1Tests defined in Workspace-App
```

would something like this be more helpful?

```
 · Cannot find targets Framework1Tests (Frameworks/Framework1_incorrect_path) defined in Workspace-App
```

Reply:
Added relative path to output